modules/renter/hostdb/consts.go

```diff
@@ -52,15 +52,15 @@ var (
 	// hostCheckupQuantity specifies the number of hosts that get scanned every
 	// time there is a regular scanning operation.
 	hostCheckupQuantity = build.Select(build.Var{
-		Standard: int(200),
+		Standard: int(10000),
```

Comment:
Should be 10e3.

How long does it take the hostdb to get through 10e3 threads? If each one takes 2 minutes and we have 50 threads, that's 6.6 hours, which I think is too long to be useful.

Should probably drop it to 2,500 unless we get confirmation from osx testing that we can jump to 200 threads. Then 10,000 is probably okay.